src/python/pants/backend/python/lint/ruff/rules.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from pants.backend.python.lint.ruff.subsystem import Ruff, RuffFieldSet
+from pants.backend.python.util_rules import pex
+from pants.backend.python.util_rules.pex import PexRequest, VenvPex, VenvPexProcess
+from pants.core.goals.fix import FixTargetsRequest
+from pants.core.goals.fmt import FmtResult
+from pants.core.util_rules.config_files import ConfigFiles, ConfigFilesRequest
+from pants.core.util_rules.partitions import PartitionerType
+from pants.engine.fs import Digest, MergeDigests
+from pants.engine.process import FallibleProcessResult
+from pants.engine.rules import Get, MultiGet, collect_rules, rule
+from pants.util.logging import LogLevel
+from pants.util.strutil import pluralize
+
+
+class RuffRequest(FixTargetsRequest):
+    field_set_type = RuffFieldSet
+    tool_subsystem = Ruff
+    partitioner_type = PartitionerType.DEFAULT_SINGLE_PARTITION
+
+
+@rule(desc="Fix with ruff", level=LogLevel.DEBUG)
+async def ruff_fix(request: RuffRequest.Batch, ruff: Ruff) -> FmtResult:
```

Comment:
```suggestion
@rule(desc="Format with ruff", level=LogLevel.DEBUG)
async def ruff_format(request: RuffRequest.Batch, ruff: Ruff) -> FmtResult:
```

Reply:
There also some other places where I'm using Fix instead of Fmt classes. I'll hold off from pushing changes for now until we make a decision on which one to use.

Reply:
@thejcannon I kept it as formatter for now since that's the least amount of change.